.github/workflows/new-issues.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: llvm/actions/issue-labeler@main
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          repo-token: ${{ secrets.ISSUE_SUBSCRIBER_TOKEN }}
```

Comment:
Is this a new least-privileged token create specifically for this workflow?

Reply:
We use this token in the new-prs.yml workflow for the same purpose.